app/views/steps/conviction/compensation_paid_amount/edit.html.erb

```diff
@@ -1,23 +1,15 @@
 <% title t('.page_title') %>
+<%= step_header %>
```

Comment:
Super minor thing, it doesn't really affect in any visible way but for the `step_header` we can now use the same syntax as with the `title` (removing the `=`) as it does not output anything in place, it just assigns the content for the `back_link`

Reply:
addressed